src/components/config-pages/header-buttons.tsx

```diff
@@ -0,0 +1,20 @@
+
+import Button from "@components/elements/button";
+import { StanfordBasicSiteSetting } from "@lib/gql/__generated__/drupal.d";
+
+type Props =
+  Omit<StanfordBasicSiteSetting, "__typename" | "id" | "metatag">
+
+
+export const HeaderButtons = ({
+  sumSiteHeaderPrim,
+  sumSiteHeaderSec,
+}: Props) => {
+      return (
+        <>
+          {sumSiteHeaderPrim && <Button href={sumSiteHeaderPrim.url} secondary>{sumSiteHeaderPrim.title}</Button>}
```

Comment:
Shouldn't the secondary button have the secondary styles?

Reply:
I would think so too but the figma mocks display it like so 
![image](https://github.com/SU-SWS/summer-nextjs/assets/34019925/44a597e1-2349-45cb-99d9-221e1c1bd675)

Reply:
We can just switch them around, make the "Primary" from Drupal be the right button. "Secondary" be the left. So it'll still work with the designs, just swapped order basically.

Reply:
My concern is around the readability and assumption our content authors will make (e.g. assuming the buttons will appear left to right, header primary then secondary button regardless of styles). 

If we are going to swap the orders, should we include help text around the styling of the button link? Or will the headings (e.g. Header Primary Button)
![image](https://github.com/SU-SWS/summer-nextjs/assets/34019925/cfffa4bd-d7ad-49bb-9c09-8e3819561b2d)

Reply:
That's ok, we can add some help text. It'll be better to keep the "Primary" link with the "Primary" styles IMO.